src/App.php

```diff
@@ -203,7 +212,8 @@ protected function configureIoc( \Slim\Helper\Set $container ) {
 	protected function configureView( View $view ) {
 		$view->replace( [
 			'app' => $this->slim,
-			'i18nCtx' => $this->slim->i18nContext
+			'i18nCtx' => $this->slim->i18nContext,
+			'supportedLanguages' => $this->getSupportedLanguages(),
```

Comment:
The mediawiki-codesniffer master doesn't complain about it, and I like doing trailing commas in multi-line arrays becuase it means the next time something gets added the diff then shows an actual meaningful change of 1 line.